classes/02class/exercises/c02-network02/fabioacp/README.md

```diff
@@ -0,0 +1,25 @@
+# c02-network02
+
+## Commands Execution Output
+
+- Answers for `10.0.0.0/16`:
+
+|Subnet  |Network        |Host Range                     |Usable Hosts                   |Broadcast      |AWS Reserved                                  |
```

Comment:
The network IPs are correct, but it is missing the netmask (CIDR notation -- missing the `/bits`)

Reply:
Added the netmask. Thanks, Daniel 👍